jmxtrans-output/jmxtrans-output-influxdb/src/main/java/com/googlecode/jmxtrans/model/output/InfluxDbWriter.java

```diff
@@ -156,8 +156,8 @@ public void doWrite(Server server, Query query, ImmutableList<Result> results) t
 
 			// send the point if filteredValues isn't empty
 			if (!filteredValues.isEmpty()) {
-				filteredValues.put("_jmx_port", Integer.parseInt(server.getPort()));
```

Comment:
Agreed that this one could be disruptive.  I removed it from the latest commit and will submit the change as a separate PR once I've re-implemented it.  I'll add a setting/flag as you mentioned to allow users to specify whether it's added as a field or a tag.